fala/templates/adviser/scotland_warning.html

```diff
@@ -0,0 +1,11 @@
+<div class="govuk-grid-column-full">
+  <div class="govuk-warning-text" id="legal_aid_in_scotland_warning_text">
+    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+    <strong class="govuk-warning-text__text">
+      <span class="govuk-warning-text__assistive">Warning </span>
+      <p>These results cover England and Wales. Find out about
+        <a class="govuk-link" href="https://www.mygov.scot/legal-aid/">Legal Aid in Scotland.</a>
```

Comment:
question - this question is more for the content designers, but should this link now open on a new tab? Otherwise you see the results and may click away from the screen.

Reply:
A good question, I can ask.

Just my thoughts, on the current region support page, we do not open in a new tab and the design provided does not have the hint text `(Opens in a new tab)` appended to it. So the ticket and design implies that open in a new tab was considered already, but no harm in asking.